mlflow/utils/virtualenv.py

```diff
@@ -221,7 +221,7 @@ def _get_virtualenv_name(python_env, work_dir_path, env_id=None):
 
 
 def _create_virtualenv(
-    local_model_path, python_bin_path, env_dir, python_env, extra_env=None, capture_output=False
+    local_model_path, python_bin_path, env_dir, python_env, extra_env=None, capture_output=True
```

Comment:
Unclear why the `capture_output` default value needs to be changed to `True`. @dbczumar to investigate

Reply:
oh I probably  used it to debug stuff and forgot to switch it back!